src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(double side, String color) {
+        name = getClass().getSimpleName();
```

Comment:
I think we can remove this logic from here(as you add it to Figure.class constructor)